definitions/v2/pornoslab.yml

```diff
@@ -56,7 +56,6 @@ search:
     exactname: ""
     replyless: 0
     replylimit: 0
-    searchdate: 0
```

Comment:
why are there a few like this with little changes moving from v1 to v2?

Reply:
https://github.com/Prowlarr/Indexers/pull/58/commits/8ff8bf21e7a62ea837bd978bd85de7244fb5b418#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R37

rather than needing to review every diff line by line to see if it's a v2 change or not; easiest to freeze off v1 updates